src/test/integration/things-test.ts

```diff
@@ -1021,6 +1035,70 @@ describe('things/', function () {
     expect(res.body[0].a).toHaveProperty('timestamp');
   });
 
+  it('should be able to subscribe to events using EventSource', async () => {
+    await addDevice(EVENT_THING);
+    if (!httpServer.address()) {
```

Comment:
Shouldn't this check be redundant? Servers are already initialized in `../common`. Is there any compelling reason to keep this initialization?

Reply:
I got in a bit of a pickle trying to get a reference to the server port, with certificate errors and then problems with jest hanging on a server that wasn't closed, so I copied this approach from websocket-util.js. But I just tried again and you're right, there's a much simpler way of doing this. Phew.